challenge_spr/README.md

```diff
@@ -0,0 +1,54 @@
+# Responsible: Sam
+
+## Testing
+
+Launch procedure:
+```
+amigo-start
+amigo-free-mode
```

Comment:
With the launch file in amigo_bringup, this changes to amigo-start & amigo-challenge-spr. This also works but should be changed